emitter/__init__.py

```diff
@@ -59,7 +58,14 @@ def Emit(self, *args):
         rooms = self._getRooms()
         extras['rooms'] = rooms if len(rooms) > 0 else ''
 
-        self._client.publish(self._key, msgpack.packb([packet, extras]))
+        self._client.publish(self._key + "#emitter", msgpack.packb([packet, extras]))
```

Comment:
This line should be deleted. 
 * for `socket.io-redis >= 0.2.0` , it is unnecessary.
 * for `socket.io-redis < 0.2.0` , socket.io will receive TWO messages (one from line61, and one from line65~68).